podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '13.0'
+platform :ios, '14.0'
 target 'ArcGIS Runtime SDK Samples' do
   use_frameworks!
   pod 'ArcGIS-Runtime-Toolkit-iOS', '~> 100.14'
```

Comment:
Should this be updated now or later?

Reply:
See my comment above, I'm unsure either. I tend to keep the actual dependency version numbers unchanged so that people who don't have access to daily build SDK can still compile from `v.next` without error.

So maybe I should not change the SDK and toolkit version in README ❓

Reply:
I think it's fine to leave as is. Updating the podfile is part of our usual release process anyway so we'll remember to update it accordingly when the time comes. Updating it now, outside of our routine, could cause confusion.